test/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
```

Comment:
What is this file for? Is it used in tests? Looks like it might be able to be removed.

Reply:
This makes a package scope in the `test` folder where all these files are treated as modules again. I just had to add one to the `src` folder as well similarly to get the tests to pass.

Reply:
(This is for Node.js execution)

Reply:
👍